bdd-generator/pom.xml

```diff
@@ -188,6 +188,12 @@
             <artifactId>spring-boot-starter-freemarker</artifactId>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework.boot</groupId>
```

Comment:
What does this dependency do and why it is marked as optional?

Reply:
I thought it would alow us to use .yml files. It works without this dependency. 
https://jakubstas.com/configuration-meta-data-in-spring-boot/#.WyijsKczaUk